1-python-wrapper/wrapper/main.py

```diff
@@ -0,0 +1,33 @@
+from flask import Flask, request
+from model.model import load_most_recent_model
+import pandas as pd
+import logging
+
+
+logger = logging.getLogger(__name__)
+app = Flask(__name__)
+
+@app.route('/predict', methods=["GET", "POST"])
+def predict():
+    if request.method == "POST":
+        print(schema)
+        input = request.json
+        data = pd.DataFrame(input, index=[0])
+        return {"classification": str(model.predict(data)[0])}
+    else:
+        return "Please provide post request according to schema"
+
+
+@app.route('/schema', methods=["GET"])
+def schema():
+    return schema
+
+
+@app.route('/metadata', methods=["GET"])
+def metadata():
+    return metadata
+
+
+if __name__=="__main__":
+    model, schema, metadata = load_most_recent_model('../models')
+    app.run(debug=True)
```

Comment:
Nitpick: no newline at the end of the file (this is generally considered basic code hygiene).

Reply:
Shall I add pre-commit-hooks for this? Or overkill for this repo?

Reply:
If we automate it, then I'd say pep8 or black.

Reply:
I have added black, flake8 and mypy

Reply:
Note guys, this is a PoC repo. I do care about standards but let's not overdo automation here 😁. Once we move the relevant code to the platform repo, obviously we're going all in.